Plugins/Public/contraband/Main.cpp

```diff
@@ -228,7 +229,44 @@ static void CheckCargo(int iClientID)
 //////////////////////////////////////////////
 bool UserCmd_ManualCargoCheck(uint iClientID, const wstring & wscCmd, const wstring & wscParam, const wchar_t * usage)
 {
-	CheckCargo(iClientID);
+	// Code duplication, but there needs to be a small change in logic to the manual check always works.
```

Comment:
no. you can make a function that takes a boolean about whether to print manual check info